src/shared/util.js

```diff
@@ -1634,7 +1635,6 @@ export {
   FormatError,
   arrayByteLength,
   arraysToBytes,
-  assert,
```

Comment:
The asserts in this file looks like internal logic, let's keep them... and export